explorer/interpreter/type_checker.cpp

```diff
@@ -4588,6 +4588,20 @@ auto TypeChecker::DeclareClassDeclaration(Nonnull<ClassDeclaration*> class_decl,
       ScopeInfo::ForClassScope(scope_info, &class_scope, std::move(bindings));
   for (Nonnull<Declaration*> m : class_decl->members()) {
     CARBON_RETURN_IF_ERROR(DeclareDeclaration(m, class_scope_info));
+
+    const auto* fun = dyn_cast<FunctionDeclaration>(m);
+    if (!fun) {
+      continue;
+    }
+
+    if (fun->virt_override() == VirtualOverride::Impl) {
+      auto vtable = (*base_class)->vtable();
```

Comment:
Seems undesirable to make a copy of the vtable here.
```suggestion
      const VTable& vtable = (*base_class)->vtable();
```